src/network/webtask.cpp

```diff
@@ -262,14 +262,14 @@ void WebTask::slotAbort() {
         m_timeoutTimerId = kInvalidTimerId;
     }
 
-    auto* const pPendingNetworkReply = m_pendingNetworkReplyWeakPtr.data();
-    QUrl requestUrl;
-
     if (m_state == State::Looping) {
         DEBUG_ASSERT(m_state == State::Looping);
-        doNetworkReplyAborted(pPendingNetworkReply);
+        doLoopingTaskAborted();
     }
 
+    auto* const pPendingNetworkReply = m_pendingNetworkReplyWeakPtr.data();
```

Comment:
Using auto here is not a good idea, because it is unclear what  pPendingNetworkReply is. 
I think we need to have a shared pointer, to receive a strong reference to the pointer.

Reply:
I think, because of I put these lines into a condition block, it looks like I added those. They were like this before this PR, I didn't change anything.
 
Is it still needed to make it a shared pointer?